message_sender/factory.py

```diff
@@ -54,11 +54,14 @@ def _raw_send(self, py_data):
             'channel_data': channel_data,
         }
 
-        data = json.dumps(py_data)
+        data = json.dumps(data)
         r = self.session.post(self.api_url, auth=self.auth,
                               data=data, headers=headers)
         r.raise_for_status()
-        return r.json()
+        res = r.json()
+        return {
+            "message_id": res.get('result', {}).get('id')
+        }
```

Comment:
Is this the only relevant key we need here? No need to provide any more info (which would probably mean needing to add mappings of keys)